yaesu/ft991.c

```diff
@@ -327,7 +327,7 @@ int ft991_set_split_mode(RIG *rig, vfo_t vfo, rmode_t tx_mode, pbwidth_t tx_widt
       {
         return err;
       }
-    strncat (restore_commands, priv->ret_data-1, NEWCAT_DATA_LEN - 1 - strlen (restore_commands));
+    strncat (restore_commands, priv->ret_data-1, NEWCAT_DATA_LEN-1);
 
```

Comment:
Hmmm.  I'm still seeing a warning here:

  CC       ft991.lo
ft991.c: In function ‘ft991_set_split_mode’:
ft991.c:330:5: warning: array subscript is above array bounds [-Warray-bounds]
     strncat (restore_commands, priv->ret_data-1, NEWCAT_DATA_LEN-1);
     ^
This is the only line it barfs this on.